feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
```

Comment:
It's really great that you've decided to add this line of code.

Reply:
Thank you for your comment, Saimir!